funnel/templates/project_comments.html.jinja2

```diff
@@ -72,6 +72,7 @@
         iscommentmoderator: {% if current_auth.user and current_auth.user.is_comment_moderator -%}true{% else %}false{% endif %},
         user: {% if current_auth.user -%}{{ { 'fullname': current_auth.user.fullname, 'avatar': current_auth.user.avatar, 'profile_url': current_auth.user.profile_url }|tojson }}{% else %}{}{% endif %},
         loginUrl: "{{ url_for('login') }}",
+        lastSeenUrl: {% if project.commentset.current_roles.subscriber %}"{{ project.commentset.url_for('update_last_seen_at') }}"{% else %}false{% endif %},
```

Comment:
`document_subscriber`. If we are changing this to just `subscriber`, a migration is necessary for previous notifications.